lua/netrw/parse.lua

```diff
@@ -52,7 +52,6 @@ M.get_node = function(line)
 	return {
 		dir = curdir,
 		node = file,
-		extension = vim.fn.fnamemodify(file, ":e"),
```

Comment:
I suggest we return the proper extension here instead of removing it from the struct. Extensions are used with custom actions; removing it would be a regression ([doc](https://github.com/prichrd/netrw.nvim/blob/master/doc/netrw.nvim.txt#L55)).

It is probably safe to assume that the first element of the dot-separated string is going to be the file name, and the rest the extension. What do you think?

Reply:
Yeah, I think you can make that assumption, I'll look into it